manifests/init.pp

```diff
@@ -103,8 +103,8 @@
 #   Override the default package provider with a specific backend to use when installing the chrony package.
 #   Also see [`package_source`](#package_source).
 # @param peers
-#   This selects the servers to use for NTP peers (symmetric association).
-#   It is an array of servers.
+#   This selects the servers to use for NTP peers. It can be an array of servers
+#   or a hash of servers to their respective options.
```

Comment:
again, this is quite confusing… and could do with an `@example`

Reply:
There is an example for `servers` higher up in the file. And the type for servers and
peers are the same.  Maybe we should just mention that?

```
#   This selects the servers to use for NTP peers. As with `servers` it can be an array of
#   servers or a hash of servers to their respective options.
```

Or perhaps we should move the example for servers to be below the description of peers and
servers?